src/i18n/strings/ar.json

```diff
@@ -1056,7 +1056,7 @@
     "Failed to upload profile picture!": "تعذَّر رفع صورة الملف الشخصي!",
     "Show more": "أظهر أكثر",
     "Show less": "أظهر أقل",
-    "Channel: %(channelName)s": "قناة: %(channelName)s",
+    "Channel: <channelLink/>": "قناة: <channelLink/>",
```

Comment:
We typically leave the translations alone to avoid the potential for conflicts in Weblate. If they do occur, it would mean burning dev / ops time to lock the translations, fiddle with all the files to resolve them, etc. I would recommend just revert all the changes to translated files.

Reply:
Sure thing, I was hoping to save time as it was just the template string but this sounds reasonable :+1: